packages/text/src/react/__specs__/heading.spec.tsx

```diff
@@ -4,13 +4,26 @@ import { axe } from 'jest-axe'
 import React from 'react'
 
 import * as stories from '../__stories__/heading.story'
+import { Heading } from '../index'
 
 describe('Text', () => {
   describe('Heading', () => {
     const cases = convertStoriesToJestCases(stories)
 
     it.todo('forwards the ref')
 
+    it('composes className', () => {
```

Comment:
With the addition of this an other specs, I think the "Class Override" stories could safely be removed.